src/Keter/Config/V10.hs

```diff
@@ -182,11 +189,20 @@ instance ParseYamlFile KeterConfig where
             <*> o .:? "env" .!= Map.empty
             <*> o .:? "connection-time-bound" .!= V04.fiveMinutes
             <*> o .:? "cli-port"
-            <*> o .:? "missing-host-response-file"
-            <*> o .:? "unknown-host-response-file"
-            <*> o .:? "proxy-exception-response-file"
+            <*> inferOptKeyFromTaggedType o
+            <*> inferOptKeyFromTaggedType o
+            <*> inferOptKeyFromTaggedType o
```

Comment:
Root cause of the mismatch is here... :eyes:  wrong order:

```haskell
            <*> o .:? "missing-host-response-file"
            <*> o .:? "unknown-host-response-file"
```

this is flipped cf. the fields of `KeterConfig`:
```haskell
    , kconfigUnknownHostResponse  :: !(Maybe F.FilePath)
    , kconfigMissingHostResponse  :: !(Maybe F.FilePath)
```

`Tagged` allows to avoid this pitfall completely.